src/ARToolkit/ARSceneView.iOS.cs

```diff
@@ -141,37 +141,41 @@ private void OnCameraDidChangeTrackingState(ARSession session, ARCamera camera)
 
         private void OnWillRenderScene(ISCNSceneRenderer renderer, SCNScene scene, double timeInSeconds)
         {
-            if (_tracking && ARSCNView != null)
+            CoreFoundation.DispatchQueue.MainQueue.DispatchAsync(() =>
             {
-                var pov = ARSCNView.PointOfView;
-                if (pov != null)
+                if (_tracking && ARSCNView != null)
                 {
-                    var c = Camera;
-                    if (c != null)
+                    var pov = ARSCNView.PointOfView;
+                    if (pov != null)
                     {
-                        var q = pov.WorldOrientation;
-                        var t = pov.Transform;
-                        if(_controller != null)
-                            _controller.TransformationMatrix = InitialTransformation + TransformationMatrix.Create(q.X, q.Y, q.Z, q.W, t.Row3.X, t.Row3.Y, t.Row3.Z);
-                        using (var frame = ARSCNView.Session.CurrentFrame)
+                        var c = Camera;
+                        if (c != null)
                         {
-                            var camera = frame?.Camera;
-                            if (camera != null)
+                            var q = pov.WorldOrientation;
+                            var t = pov.WorldPosition;
+                            if (_controller != null)
+                                _controller.TransformationMatrix = InitialTransformation + TransformationMatrix.Create(q.X, q.Y, q.Z, q.W, t.X, t.Y, t.Z);
```

Comment:
All this says "Fix issue with AR position on iOS" in commit. I'm not quite following why this change is needed. See commit https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/pull/466/commits/f24a3850d28f74ba35cf7896747e78672e19742a